flutter/coffeeapp/assets/brewingsMock.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
@mmiask  to może być nasz wspólny kontrakt. Np. taką formę JSONa możemy trzymać na backendzie